internal/dcache/metadata_manager/metadata_manager_impl.go

```diff
@@ -321,6 +339,11 @@ func (m *BlobMetadataManager) createFileInit(filePath string, fileMetadata []byt
 		EtagMatchConditions:    "",
 	})
 
+	injectErr := common.InjectError(common.PROB_MODERATE)
+	if injectErr != nil {
+		err = injectErr
+	}
```

Comment:
this will cause the assert at L364 to fire.
We need to make sure that when we inject error, there are no asserts in that entire code path